static/js/settings_account.js

```diff
@@ -63,7 +63,7 @@ exports.set_up = function () {
     $("form.your-account-settings").ajaxForm({
         dataType: 'json', // This seems to be ignored. We still get back an xhr.
         beforeSubmit: function () {
-            if (page_params.password_auth_enabled !== false) {
+            if (page_params.realm_password_auth_enabled !== false) {
```

Comment:
You missed a similar reference slightly earlier in this file; I just went ahead and fixed it.

(Found via `git grep password_auth_enabled static/js`)